PowerShell.sublime-syntax

```diff
@@ -186,7 +186,7 @@ contexts:
       captures:
         1: punctuation.definition.keyword.powershell
     # Flags/Options/Parameters
-    - match: \B([-/])\p{L}(?:[\w-]*\w)?
+    - match: \B(-|--|/|\+)\p{L}(?:[\w-]*\w)?
```

Comment:
Seems fine. You can also cut this down to `(--|[-/+])` or `(--?|[/+])`. Not a big deal though, and I guess some people like all the alternatives expanded.

Reply:
Thanks for the feedback! I like the latter option you provided, because it saves the reader from having to decode the regex escaping and is nice and compact.